tests/test_validate_expected_origin.py

```diff
@@ -0,0 +1,61 @@
+from unittest import TestCase
+
+from webauthn.helpers.validate_expected_origin import (
+    match_origins,
+    validate_expected_origin,
+)
+
+
+class TestValidateExpectedOrigin(TestCase):
+
+
+    def test_match_origins(self):
+        """Test for combined match (exact and wildcard)."""
+        expected_origin_result = [
```

Comment:
Can you please add some tests to ensure that this logic will work with `localhost` addresses too please? A few obvious local dev addresses come to mind:

- `http://localhost`
- `http://localhost:8000`
- `https://localhost` ([some devs use mkcert](https://web.dev/articles/how-to-use-local-https) to host localhost over TLS)

I _really_ don't want this to impact the (read: my 😂) ability to develop on localhost.